10up/windows-azure-storage
# add spanish translation

### Description of the Change
This PR adds the Spanish (`es_ES`) translation for the plugin

### How to test the Change

To try this
- Pull down the branch
- Go to Dashboard > Settings > General
- Switch the WordPress language to "Español ( España )"  a.k.a `es_ES`
- Check the plugin configuration: Dashboard > Settings > Microsoft Azure

![Screenshot 2024-07-04 at 16 18 05](https://github.com/10up/windows-azure-storage/assets/894708/33b5129a-e03f-4c29-b63c-0ae70156f862)

### Changelog Entry

> Added - New feature
Spanish (es_ES) translation


### Credits
Props @hugosolar 


### Checklist:
